README.md
# The Gathering Storm - A Wheel of Time Mod (Beta)
## “The Wheel of Time turns...”
The blow text is almost entirely copied from [this thread on the Taleworlds forums](https://forums.taleworlds.com/index.php/topic,147665.0.html).

The Gathering Storm is a mod based on Robert Jordan's Wheel of Time series.  It strives to capture the essence of the epic fantasy tale, while also retaining the open-ended feel of Mount and Blade.

This is a Beta version, meaning there are still hundreds of areas that can be improved upon.  Please try it out and share any feedback.

The main developers of this project (mat2rivs, vaerraent) are on hiatus for the forseeable future.

See ``license.txt`` for license information, ``installation.txt`` for installation information, ``changelog.txt`` for a changelog and ``CONTRIBUTING.md`` for information on how to contribute. 

## Feature List:
* Wheel of Time Map - Expanded in 0.2.7 to include Seanchan, Sea Folk Isles, Shara, and the Land of Madmen
* Twenty-eight factions including all nations and city states, plus several other you should recognize
* An extensive troop tree including all political regions that appear in the series.
* 170days of forced diplomacy that follows the general events in the series.  After 170 days, diplomacy reverts to generic Mount and Blade.
* Aes Sedai will spawn warders. Sul'dam spawn Damane.  Asha'man spawn captured Aes Sedai.
* Several kits from the forum community. (troop ratio bar, Auto-loot, and Recruiter Kit)
* Press 'H' when in the world map view for a help screen.
* Two new custom battle stages based on the Wheel of Time world.
* Integration of Diplomacy 4.2 with PBOD 0.90.1  (for Warband v1.143)
* Naval travel and warfare. (not menu based)
* Channeling System that includes 14 unique weaves.
* Intuitive weave toggling method with configurable keys.
* NPC channelers have AI scripts that help them choose logical weaves based on their disposition and also the current situation on the battlefield.
* The strength of known weaves scales as your character advances in the Five-Power skills (Air, Water, Spirit, Fire, and Earth)

## Channeling System Basics:
All weaves take stamina to channel.  A HUD bar will show how much stamina is remaining.  Stamina regenerates based on Intelligence and Channeling Proficiency, and has a maximum amount based on the Channeling Affinity skill, Intelligence, and Channeling Proficiency.  You can switch between weaves by pressing “Caps-Lock” and using the mouse to choose your weave.  To see what playing as a channeler is like, try a Custom/Quick Battle using Rand or Nynaeve as your character.  To be a channeler in the Single Player Campaign, be sure to choose the childhood option of 'Novice or Wilder' if you are female, or 'Wilder' if you are male.

## A detailed explanation of the 14 weaves:
* Air blast - (Visual - Dust Cloud) does a little damage and pushes enemies (push effect only for single player open field battles)
* Freeze - (Visual - ice trail) slows enemies who are hit and deals minimal damage
* Heal - (Visual - purple mist on the agent you healed) heals nearest allied agent with less that 70% health (can heal your horse)
* Fireball - (Visual - fire trail) does initial damage and them burn over time damage on all agents in target zone
* Unravel - (Visual - yellow mist on effect that was unraveled) Dispels seekers, compulsion, bind, and allies who are burning
* Defensive blast - (Visual - Dust Cloud) does large damage to agents who are near channeler (and pushes them away if you are in single player open field battles)
* Ranged blast - (Visual - cloud of dust on impact) Sort of a ranged version of the previous weave
* Bind - (Visual - white mist on bound agent) bound agent can't move, can be damaged
* Chain Lightening - (Visual - Lightning strike from the skies) Will damage nearby agents and pass charge on to others
* Fire Curtain - (Visual - wall of fire) Will do initial damage and damage over time to all agents near the fire wall, burns for set duration
* Shield - (Visual - purple mist on agent who is shielded) shielded agent can't use channeling unless they break free.  If you are shielded keep channeling and you have a chance of breaking free.  Harder to do if a strong enemy shielded you.
* Seeker - (Visual - ball of green energy that tracks and kills target) You get hit, you die, also does AOE damage at higher levels
* Compulsion - (Visual - black mist on compelled agent) This will turn the compelled enemy agent to your team.
* Balefire - (Visual - light trail and purple image if agent is hit)  Agents in the path of the balefire stream will be ripped from the Pattern and their past actions erased. (killed instantly and removed from your sight) Also if the killed agent has been killing your teammates, then some of them may come back to life.

## Installation
Either download the latest version from [Nexus](http://www.nexusmods.com/mbwarband/mods/3487) (be sure to download the 0.2.7d update file, too) or, for the latest development version, 'Download as ZIP' or clone this repository. This repository does not include resources such as models, textures and sounds - these should be downloaded separately, from [this link](https://drive.google.com/file/d/0B3jrGydxOpsQeGFxMFRQWjZVX1U/view?usp=sharing).

## Contributing
Please [fork](https://help.github.com/articles/fork-a-repo/) the project and create [pull requests](https://help.github.com/articles/using-pull-requests/) for changes.  


## Notes
* Battles with large number of channelers can cause lagging or even crashing.  Also, if you are using AI formation from PBOD, or also special actions such as AI spear bracing, this can cause lagging as well.  Take some time to tune your options to fit the capabilities of your computer.

This mod is compatible with v1.143 of Warband.

This mod will play similarly to Feral's mod “Shadow Rising”.  However, I did not use any of his models or files per his request.

The Wheel of Time is © Copyright Robert Jordan.
The phrases The Wheel of Time™ and The Dragon Reborn™ are trademarks of Robert Jordan.


changelog.txt

V0.2.7d Beta Hotfix
Updates:
-The Malden scene is now completed in multiplayer mode
-The completed Malden scene has now been implemented in the single player campaign

V0.2.7c Beta Hotfix
Fixes:
-Starting as a Noble and chosing a faction's banner should no longer affect that faction's banner on the main map
-Seekers and Firewalls should not carry over from battle to battle
-Added additional error fixing measures for the Channeling stamina error
-Added Weave Select capabilities for the Ways
-Fixed Seanchan to Altara/Tarabon/Amadicia diplomacy issues from days 95 to 118
-All war/peace declarations that don't affect the player's kingdom should be 'silent'
-Fixed the issue where the 'channeling proficiency gained due to use in battle' message fired continually
-Fixed the troop ratio bar and channeling stamina presentation so it works better with other presentations such as 'View Orders', formations, etc
-Permanently disabled Defecting Lords (unless they are scripted) and Randomly Generated Conflicts so the endgame won't be as chaotic

V0.2.7b Beta Hotfix
Fixes:
-Rescue Merchant's Nephew errors
 -No Tavernkeepers in some towns error
 
V0.2.7a Beta Hotfix
Fixes:
-Infinite channeling stamina error
 -Sul'dam / Damane script error
 -Minor Channeling Stamina weave usage tweaks

V0.2.7 Beta Full
Fixes:
-Fixed mixed up Companion stuff
 -Cleaned up non-intended relationships from Diplomacy mod
 -Fixed error where Kingdom ladies didn't have clothing items
 -Fixed error where players who picked 'Male', 'Wilder' were spawned wearing a dress
 -Fixed error where lords are not equipping stuff at feasts
 -Fixed Faction Relation Presentation - (updated with the latest code from Jrider)
 -Fixed mst_village_attack_bandit error (I hope...)
 -Corrected starting tutorial Caravan destination strings for split factions.
 -Renamed remaining Native Castles
 -Fixed 'Rahvin/Gaebril' naming error for the Lady Arymilla pretender dialogs
 -Gave lancers side arms
 -Many other minor bug fixes.
 
Updated Features:
-Ported to Diplomacy 4.2 and PBOD 0.90.1
 -Updated map to add more territories (expanded Blight, Seanchan, Shara, Madmen, Sea Folk Isles)
 -Added water travel and sea battles
 -Added border towers that will cause a battle nearby to use the Shienar Border Tower scene
 -Split the existing eight factions into 24 nation based factions
 -Added four new factions for Shara, Sea Folk, Madmen, and Toman Head
 -Added Shara outfits
 -Added Sea Folk outfits
 -Added some painted body skins for the Madmen
 -Gave all lords a random 'reputation' - will need to re-visit and make it specific for the main characters
 -Overhauled channeling system to make it modular and scalable
 -Added Skills for Channeling Affinity and the Five Powers
 -Re-wrote channeling stamina equation (Channeling stamina will no longer re-fill after every battle)
 -NPC channelers now have AI that governs which weaves they use
 -Channeling companions will no longer use the Primary/Secondary/All weave selection process (They will use the AI instead.)
 -Created a new method for weave selection (It's now a 'point and click' method)

ID_scenes.py
scn_random_scene = 0
scn_conversation_scene = 1
scn_water = 2
scn_random_scene_steppe = 3
scn_random_scene_plain = 4
scn_random_scene_snow = 5
scn_random_scene_desert = 6
scn_random_scene_steppe_forest = 7
scn_random_scene_plain_forest = 8
scn_random_scene_snow_forest = 9
scn_random_scene_desert_forest = 10
scn_camp_scene = 11
scn_camp_scene_horse_track = 12
scn_four_ways_inn = 13
scn_test_scene = 14
scn_quick_battle_1 = 15
scn_quick_battle_2 = 16
scn_quick_battle_3 = 17
scn_quick_battle_4 = 18
scn_quick_battle_5 = 19
scn_quick_battle_6 = 20
scn_quick_battle_7 = 21
scn_salt_mine = 22
scn_novice_ground = 23
scn_zendar_arena = 24
scn_dhorak_keep = 25
scn_reserved4 = 26
scn_reserved5 = 27
scn_reserved6 = 28
scn_reserved7 = 29
scn_reserved8 = 30
scn_reserved9 = 31
scn_reserved10 = 32
scn_reserved11 = 33
scn_reserved12 = 34
scn_training_ground = 35
scn_tutorial_1 = 36
scn_tutorial_2 = 37
scn_tutorial_3 = 38
scn_tutorial_4 = 39
scn_tutorial_5 = 40
scn_ways = 41
scn_training_ground_horse_track_1 = 42
scn_training_ground_horse_track_2 = 43
scn_training_ground_horse_track_3 = 44
scn_training_ground_horse_track_4 = 45
scn_training_ground_horse_track_5 = 46
scn_training_ground_ranged_melee_1 = 47
scn_training_ground_ranged_melee_2 = 48
scn_training_ground_ranged_melee_3 = 49
scn_training_ground_ranged_melee_4 = 50
scn_training_ground_ranged_melee_5 = 51
scn_zendar_center = 52
scn_the_happy_boar = 53
scn_zendar_merchant = 54
scn_town_1_center = 55
scn_town_2_center = 56
scn_town_3_center = 57
scn_town_4_center = 58
scn_town_5_center = 59
scn_town_6_center = 60
scn_town_7_center = 61
scn_town_8_center = 62
scn_town_9_center = 63
scn_town_10_center = 64
scn_town_11_center = 65
scn_town_12_center = 66
scn_town_13_center = 67
scn_town_14_center = 68
scn_town_15_center = 69
scn_town_16_center = 70
scn_town_17_center = 71
scn_town_18_center = 72
scn_town_19_center = 73
scn_town_20_center = 74
scn_town_21_center = 75
scn_town_22_center = 76
scn_town_23_center = 77
scn_town_24_center = 78
scn_town_25_center = 79
scn_town_26_center = 80
scn_town_27_center = 81
scn_town_28_center = 82
scn_town_29_center = 83
scn_town_30_center = 84
scn_town_31_center = 85
scn_town_32_center = 86
scn_town_33_center = 87
scn_town_34_center = 88
scn_town_35_center = 89
scn_town_36_center = 90
scn_town_37_center = 91
scn_town_38_center = 92
scn_town_1_castle = 93
scn_town_2_castle = 94
scn_town_3_castle = 95
scn_town_4_castle = 96
scn_town_5_castle = 97
scn_town_6_castle = 98
scn_town_7_castle = 99
scn_town_8_castle = 100
scn_town_9_castle = 101
scn_town_10_castle = 102
scn_town_11_castle = 103
scn_town_12_castle = 104
scn_town_13_castle = 105
scn_town_14_castle = 106
scn_town_15_castle = 107
scn_town_16_castle = 108
scn_town_17_castle = 109
scn_town_18_castle = 110
scn_town_19_castle = 111
scn_town_20_castle = 112
scn_town_21_castle = 113
scn_town_22_castle = 114
scn_town_23_castle = 115
scn_town_24_castle = 116
scn_town_25_castle = 117
scn_town_26_castle = 118
scn_town_27_castle = 119
scn_town_28_castle = 120
scn_town_29_castle = 121
scn_town_30_castle = 122
scn_town_31_castle = 123
scn_town_32_castle = 124
scn_town_33_castle = 125
scn_town_34_castle = 126
scn_town_35_castle = 127
scn_town_36_castle = 128
scn_town_37_castle = 129
scn_town_38_castle = 130
scn_town_1_tavern = 131
scn_town_2_tavern = 132
scn_town_3_tavern = 133
scn_town_4_tavern = 134
scn_town_5_tavern = 135
scn_town_6_tavern = 136
scn_town_7_tavern = 137
scn_town_8_tavern = 138
scn_town_9_tavern = 139
scn_town_10_tavern = 140
scn_town_11_tavern = 141
scn_town_12_tavern = 142
scn_town_13_tavern = 143
scn_town_14_tavern = 144
scn_town_15_tavern = 145
scn_town_16_tavern = 146
scn_town_17_tavern = 147
scn_town_18_tavern = 148
scn_town_19_tavern = 149
scn_town_20_tavern = 150
scn_town_21_tavern = 151
scn_town_22_tavern = 152
scn_town_23_tavern = 153
scn_town_24_tavern = 154
scn_town_25_tavern = 155
scn_town_26_tavern = 156
scn_town_27_tavern = 157
scn_town_28_tavern = 158
scn_town_29_tavern = 159
scn_town_30_tavern = 160
scn_town_31_tavern = 161
scn_town_32_tavern = 162
scn_town_33_tavern = 163
scn_town_34_tavern = 164
scn_town_35_tavern = 165
scn_town_36_tavern = 166
scn_town_37_tavern = 167
scn_town_38_tavern = 168
scn_town_1_store = 169
scn_town_2_store = 170
scn_town_3_store = 171
scn_town_4_store = 172
scn_town_5_store = 173
scn_town_6_store = 174
scn_town_7_store = 175
scn_town_8_store = 176
scn_town_9_store = 177
scn_town_10_store = 178
scn_town_11_store = 179
scn_town_12_store = 180
scn_town_13_store = 181
scn_town_14_store = 182
scn_town_15_store = 183
scn_town_16_store = 184
scn_town_17_store = 185
scn_town_18_store = 186
scn_town_19_store = 187
scn_town_20_store = 188
scn_town_21_store = 189
scn_town_22_store = 190
scn_town_23_store = 191
scn_town_24_store = 192
scn_town_25_store = 193
scn_town_26_store = 194
scn_town_27_store = 195
scn_town_28_store = 196
scn_town_29_store = 197
scn_town_30_store = 198
scn_town_31_store = 199
scn_town_32_store = 200
scn_town_33_store = 201
scn_town_34_store = 202
scn_town_35_store = 203
scn_town_36_store = 204
scn_town_37_store = 205
scn_town_38_store = 206
scn_town_1_arena = 207
scn_town_2_arena = 208
scn_town_3_arena = 209
scn_town_4_arena = 210
scn_town_5_arena = 211
scn_town_6_arena = 212
scn_town_7_arena = 213
scn_town_8_arena = 214
scn_town_9_arena = 215
scn_town_10_arena = 216
scn_town_11_arena = 217
scn_town_12_arena = 218
scn_town_13_arena = 219
scn_town_14_arena = 220
scn_town_15_arena = 221
scn_town_16_arena = 222
scn_town_17_arena = 223
scn_town_18_arena = 224
scn_town_19_arena = 225
scn_town_20_arena = 226
scn_town_21_arena = 227
scn_town_22_arena = 228
scn_town_23_arena = 229
scn_town_24_arena = 230
scn_town_25_arena = 231
scn_town_26_arena = 232
scn_town_27_arena = 233
scn_town_28_arena = 234
scn_town_29_arena = 235
scn_town_30_arena = 236
scn_town_31_arena = 237
scn_town_32_arena = 238
scn_town_33_arena = 239
scn_town_34_arena = 240
scn_town_35_arena = 241
scn_town_36_arena = 242
scn_town_37_arena = 243
scn_town_38_arena = 244
scn_town_1_prison = 245
scn_town_2_prison = 246
scn_town_3_prison = 247
scn_town_4_prison = 248
scn_town_5_prison = 249
scn_town_6_prison = 250
scn_town_7_prison = 251
scn_town_8_prison = 252
scn_town_9_prison = 253
scn_town_10_prison = 254
scn_town_11_prison = 255
scn_town_12_prison = 256
scn_town_13_prison = 257
scn_town_14_prison = 258
scn_town_15_prison = 259
scn_town_16_prison = 260
scn_town_17_prison = 261
scn_town_18_prison = 262
scn_town_19_prison = 263
scn_town_20_prison = 264
scn_town_21_prison = 265
scn_town_22_prison = 266
scn_town_23_prison = 267
scn_town_24_prison = 268
scn_town_25_prison = 269
scn_town_26_prison = 270
scn_town_27_prison = 271
scn_town_28_prison = 272
scn_town_29_prison = 273
scn_town_30_prison = 274
scn_town_31_prison = 275
scn_town_32_prison = 276
scn_town_33_prison = 277
scn_town_34_prison = 278
scn_town_35_prison = 279
scn_town_36_prison = 280
scn_town_37_prison = 281
scn_town_38_prison = 282
scn_town_1_walls = 283
scn_town_2_walls = 284
scn_town_3_walls = 285
scn_town_4_walls = 286
scn_town_5_walls = 287
scn_town_6_walls = 288
scn_town_7_walls = 289
scn_town_8_walls = 290
scn_town_9_walls = 291
scn_town_10_walls = 292
scn_town_11_walls = 293
scn_town_12_walls = 294
scn_town_13_walls = 295
scn_town_14_walls = 296
scn_town_15_walls = 297
scn_town_16_walls = 298
scn_town_17_walls = 299
scn_town_18_walls = 300
scn_town_19_walls = 301
scn_town_20_walls = 302
scn_town_21_walls = 303
scn_town_22_walls = 304
scn_town_23_walls = 305
scn_town_24_walls = 306
scn_town_25_walls = 307
scn_town_26_walls = 308
scn_town_27_walls = 309
scn_town_28_walls = 310
scn_town_29_walls = 311
scn_town_30_walls = 312
scn_town_31_walls = 313
scn_town_32_walls = 314
scn_town_33_walls = 315
scn_town_34_walls = 316
scn_town_35_walls = 317
scn_town_36_walls = 318
scn_town_37_walls = 319
scn_town_38_walls = 320
scn_town_1_alley = 321
scn_town_2_alley = 322
scn_town_3_alley = 323
scn_town_4_alley = 324
scn_town_5_alley = 325
scn_town_6_alley = 326
scn_town_7_alley = 327
scn_town_8_alley = 328
scn_town_9_alley = 329
scn_town_10_alley = 330
scn_town_11_alley = 331
scn_town_12_alley = 332
scn_town_13_alley = 333
scn_town_14_alley = 334
scn_town_15_alley = 335
scn_town_16_alley = 336
scn_town_17_alley = 337
scn_town_18_alley = 338
scn_town_19_alley = 339
scn_town_20_alley = 340
scn_town_21_alley = 341
scn_town_22_alley = 342
scn_town_23_alley = 343
scn_town_24_alley = 344
scn_town_25_alley = 345
scn_town_26_alley = 346
scn_town_27_alley = 347
scn_town_28_alley = 348
scn_town_29_alley = 349
scn_town_30_alley = 350
scn_town_31_alley = 351
scn_town_32_alley = 352
scn_town_33_alley = 353
scn_town_34_alley = 354
scn_town_35_alley = 355
scn_town_36_alley = 356
scn_town_37_alley = 357
scn_town_38_alley = 358
scn_castle_1_exterior = 359
scn_castle_1_interior = 360
scn_castle_1_prison = 361
scn_castle_2_exterior = 362
scn_castle_2_interior = 363
scn_castle_2_prison = 364
scn_castle_3_exterior = 365
scn_castle_3_interior = 366
scn_castle_3_prison = 367
scn_castle_4_exterior = 368
scn_castle_4_interior = 369
scn_castle_4_prison = 370
scn_castle_5_exterior = 371
scn_castle_5_interior = 372
scn_castle_5_prison = 373
scn_castle_6_exterior = 374
scn_castle_6_interior = 375
scn_castle_6_prison = 376
scn_castle_7_exterior = 377
scn_castle_7_interior = 378
scn_castle_7_prison = 379
scn_castle_8_exterior = 380
scn_castle_8_interior = 381
scn_castle_8_prison = 382
scn_castle_9_exterior = 383
scn_castle_9_interior = 384
scn_castle_9_prison = 385
scn_castle_10_exterior = 386
scn_castle_10_interior = 387
scn_castle_10_prison = 388
scn_castle_11_exterior = 389
scn_castle_11_interior = 390
scn_castle_11_prison = 391
scn_castle_12_exterior = 392
scn_castle_12_interior = 393
scn_castle_12_prison = 394
scn_castle_13_exterior = 395
scn_castle_13_interior = 396
scn_castle_13_prison = 397
scn_castle_14_exterior = 398
scn_castle_14_interior = 399
scn_castle_14_prison = 400
scn_castle_15_exterior = 401
scn_castle_15_interior = 402
scn_castle_15_prison = 403
scn_castle_16_exterior = 404
scn_castle_16_interior = 405
scn_castle_16_prison = 406
scn_castle_17_exterior = 407
scn_castle_17_interior = 408
scn_castle_17_prison = 409
scn_castle_18_exterior = 410
scn_castle_18_interior = 411
scn_castle_18_prison = 412
scn_castle_19_exterior = 413
scn_castle_19_interior = 414
scn_castle_19_prison = 415
scn_castle_20_exterior = 416
scn_castle_20_interior = 417
scn_castle_20_prison = 418
scn_castle_21_exterior = 419
scn_castle_21_interior = 420
scn_castle_21_prison = 421
scn_castle_22_exterior = 422
scn_castle_22_interior = 423
scn_castle_22_prison = 424
scn_castle_23_exterior = 425
scn_castle_23_interior = 426
scn_castle_23_prison = 427
scn_castle_24_exterior = 428
scn_castle_24_interior = 429
scn_castle_24_prison = 430
scn_castle_25_exterior = 431
scn_castle_25_interior = 432
scn_castle_25_prison = 433
scn_castle_26_exterior = 434
scn_castle_26_interior = 435
scn_castle_26_prison = 436
scn_castle_27_exterior = 437
scn_castle_27_interior = 438
scn_castle_27_prison = 439
scn_castle_28_exterior = 440
scn_castle_28_interior = 441
scn_castle_28_prison = 442
scn_castle_29_exterior = 443
scn_castle_29_interior = 444
scn_castle_29_prison = 445
scn_castle_30_exterior = 446
scn_castle_30_interior = 447
scn_castle_30_prison = 448
scn_castle_31_exterior = 449
scn_castle_31_interior = 450
scn_castle_31_prison = 451
scn_castle_32_exterior = 452
scn_castle_32_interior = 453
scn_castle_32_prison = 454
scn_castle_33_exterior = 455
scn_castle_33_interior = 456
scn_castle_33_prison = 457
scn_castle_34_exterior = 458
scn_castle_34_interior = 459
scn_castle_34_prison = 460
scn_castle_35_exterior = 461
scn_castle_35_interior = 462
scn_castle_35_prison = 463
scn_castle_36_exterior = 464
scn_castle_36_interior = 465
scn_castle_36_prison = 466
scn_castle_37_exterior = 467
scn_castle_37_interior = 468
scn_castle_37_prison = 469
scn_castle_38_exterior = 470
scn_castle_38_interior = 471
scn_castle_38_prison = 472
scn_castle_39_exterior = 473
scn_castle_39_interior = 474
scn_castle_39_prison = 475
scn_castle_40_exterior = 476
scn_castle_40_interior = 477
scn_castle_40_prison = 478
scn_castle_41_exterior = 479
scn_castle_41_interior = 480
scn_castle_41_prison = 481
scn_castle_42_exterior = 482
scn_castle_42_interior = 483
scn_castle_42_prison = 484
scn_castle_43_exterior = 485
scn_castle_43_interior = 486
scn_castle_43_prison = 487
scn_castle_44_exterior = 488
scn_castle_44_interior = 489
scn_castle_44_prison = 490
scn_castle_45_exterior = 491
scn_castle_45_interior = 492
scn_castle_45_prison = 493
scn_castle_46_exterior = 494
scn_castle_46_interior = 495
scn_castle_46_prison = 496
scn_castle_47_exterior = 497
scn_castle_47_interior = 498
scn_castle_47_prison = 499
scn_castle_48_exterior = 500
scn_castle_48_interior = 501
scn_castle_48_prison = 502
scn_castle_49_exterior = 503
scn_castle_49_interior = 504
scn_castle_49_prison = 505
scn_castle_50_exterior = 506
scn_castle_50_interior = 507
scn_castle_50_prison = 508
scn_castle_51_exterior = 509
scn_castle_51_interior = 510
scn_castle_51_prison = 511
scn_castle_52_exterior = 512
scn_castle_52_interior = 513
scn_castle_52_prison = 514
scn_castle_53_exterior = 515
scn_castle_53_interior = 516
scn_castle_53_prison = 517
scn_castle_54_exterior = 518
scn_castle_54_interior = 519
scn_castle_54_prison = 520
scn_castle_55_exterior = 521
scn_castle_55_interior = 522
scn_castle_55_prison = 523
scn_castle_56_exterior = 524
scn_castle_56_interior = 525
scn_castle_56_prison = 526
scn_castle_57_exterior = 527
scn_castle_57_interior = 528
scn_castle_57_prison = 529
scn_castle_58_exterior = 530
scn_castle_58_interior = 531
scn_castle_58_prison = 532
scn_castle_59_exterior = 533
scn_castle_59_interior = 534
scn_castle_59_prison = 535
scn_castle_60_exterior = 536
scn_castle_60_interior = 537
scn_castle_60_prison = 538
scn_castle_61_exterior = 539
scn_castle_61_interior = 540
scn_castle_61_prison = 541
scn_castle_62_exterior = 542
scn_castle_62_interior = 543
scn_castle_62_prison = 544
scn_castle_63_exterior = 545
scn_castle_63_interior = 546
scn_castle_63_prison = 547
scn_castle_64_exterior = 548
scn_castle_64_interior = 549
scn_castle_64_prison = 550
scn_castle_65_exterior = 551
scn_castle_65_interior = 552
scn_castle_65_prison = 553
scn_castle_66_exterior = 554
scn_castle_66_interior = 555
scn_castle_66_prison = 556
scn_castle_67_exterior = 557
scn_castle_67_interior = 558
scn_castle_67_prison = 559
scn_castle_68_exterior = 560
scn_castle_68_interior = 561
scn_castle_68_prison = 562
scn_castle_69_exterior = 563
scn_castle_69_interior = 564
scn_castle_69_prison = 565
scn_castle_70_exterior = 566
scn_castle_70_interior = 567
scn_castle_70_prison = 568
scn_village_1 = 569
scn_village_2 = 570
scn_village_3 = 571
scn_village_4 = 572
scn_village_5 = 573
scn_village_6 = 574
scn_village_7 = 575
scn_village_8 = 576
scn_village_9 = 577
scn_village_10 = 578
scn_village_11 = 579
scn_village_12 = 580
scn_village_13 = 581
scn_village_14 = 582
scn_village_15 = 583
scn_village_16 = 584
scn_village_17 = 585
scn_village_18 = 586
scn_village_19 = 587
scn_village_20 = 588
scn_village_21 = 589
scn_village_22 = 590
scn_village_23 = 591
scn_village_24 = 592
scn_village_25 = 593
scn_village_26 = 594
scn_village_27 = 595
scn_village_28 = 596
scn_village_29 = 597
scn_village_30 = 598
scn_village_31 = 599
scn_village_32 = 600
scn_village_33 = 601
scn_village_34 = 602
scn_village_35 = 603
scn_village_36 = 604
scn_village_37 = 605
scn_village_38 = 606
scn_village_39 = 607
scn_village_40 = 608
scn_village_41 = 609
scn_village_42 = 610
scn_village_43 = 611
scn_village_44 = 612
scn_village_45 = 613
scn_village_46 = 614
scn_village_47 = 615
scn_village_48 = 616
scn_village_49 = 617
scn_village_50 = 618
scn_village_51 = 619
scn_village_52 = 620
scn_village_53 = 621
scn_village_54 = 622
scn_village_55 = 623
scn_village_56 = 624
scn_village_57 = 625
scn_village_58 = 626
scn_village_59 = 627
scn_village_60 = 628
scn_village_61 = 629
scn_village_62 = 630
scn_village_63 = 631
scn_village_64 = 632
scn_village_65 = 633
scn_village_66 = 634
scn_village_67 = 635
scn_village_68 = 636
scn_village_69 = 637
scn_village_70 = 638
scn_village_71 = 639
scn_village_72 = 640
scn_village_73 = 641
scn_village_74 = 642
scn_village_75 = 643
scn_village_76 = 644
scn_village_77 = 645
scn_village_78 = 646
scn_village_79 = 647
scn_village_80 = 648
scn_village_81 = 649
scn_village_82 = 650
scn_village_83 = 651
scn_village_84 = 652
scn_village_85 = 653
scn_village_86 = 654
scn_village_87 = 655
scn_village_88 = 656
scn_village_89 = 657
scn_village_90 = 658
scn_village_91 = 659
scn_village_92 = 660
scn_village_93 = 661
scn_village_94 = 662
scn_village_95 = 663
scn_village_96 = 664
scn_village_97 = 665
scn_village_98 = 666
scn_village_99 = 667
scn_village_100 = 668
scn_village_101 = 669
scn_village_102 = 670
scn_village_103 = 671
scn_village_104 = 672
scn_village_105 = 673
scn_village_106 = 674
scn_village_107 = 675
scn_village_108 = 676
scn_village_109 = 677
scn_village_110 = 678
scn_village_111 = 679
scn_village_112 = 680
scn_village_113 = 681
scn_village_114 = 682
scn_village_115 = 683
scn_village_116 = 684
scn_village_117 = 685
scn_village_118 = 686
scn_village_119 = 687
scn_village_120 = 688
scn_village_121 = 689
scn_village_122 = 690
scn_village_123 = 691
scn_village_124 = 692
scn_village_125 = 693
scn_village_126 = 694
scn_village_127 = 695
scn_village_128 = 696
scn_village_129 = 697
scn_village_130 = 698
scn_village_131 = 699
scn_village_132 = 700
scn_village_133 = 701
scn_village_134 = 702
scn_village_135 = 703
scn_village_136 = 704
scn_village_137 = 705
scn_village_138 = 706
scn_village_139 = 707
scn_village_140 = 708
scn_village_141 = 709
scn_village_142 = 710
scn_village_143 = 711
scn_village_144 = 712
scn_village_145 = 713
scn_village_146 = 714
scn_village_147 = 715
scn_village_148 = 716
scn_village_149 = 717
scn_village_150 = 718
scn_village_151 = 719
scn_field_1 = 720
scn_field_2 = 721
scn_field_3 = 722
scn_field_4 = 723
scn_field_5 = 724
scn_test2 = 725
scn_test3 = 726
scn_multi_scene_1 = 727
scn_multi_scene_2 = 728
scn_multi_scene_3 = 729
scn_multi_scene_4 = 730
scn_multi_scene_5 = 731
scn_multi_scene_6 = 732
scn_multi_scene_7 = 733
scn_multi_scene_8 = 734
scn_multi_scene_9 = 735
scn_multi_scene_10 = 736
scn_multi_scene_11 = 737
scn_multi_scene_12 = 738
scn_multi_scene_13 = 739
scn_multi_scene_14 = 740
scn_multi_scene_15 = 741
scn_multi_scene_16 = 742
scn_multi_scene_17 = 743
scn_multi_scene_18 = 744
scn_random_multi_plain_medium = 745
scn_random_multi_plain_large = 746
scn_random_multi_steppe_medium = 747
scn_random_multi_steppe_large = 748
scn_multi_emonds_field = 749
scn_multi_shienaran_border_tower = 750
scn_multi_malden = 751
scn_multi_ways = 752
scn_multi_sea_b = 753
scn_multi_sea_land = 754
scn_multi_ship_battle = 755
scn_multi_ship_landing = 756
scn_multi_blank_desert_small = 757
scn_multi_blank_desert_medium = 758
scn_multi_blank_desert_large = 759
scn_multi_blank_desert_huge = 760
scn_multi_blank_snow_small = 761
scn_multi_blank_snow_medium = 762
scn_multi_blank_snow_large = 763
scn_multi_blank_snow_huge = 764
scn_multi_blank_plain_small = 765
scn_multi_blank_plain_medium = 766
scn_multi_blank_plain_large = 767
scn_multi_blank_plain_huge = 768
scn_multi_blank_steppe_small = 769
scn_multi_blank_steppe_medium = 770
scn_multi_blank_steppe_large = 771
scn_multi_blank_steppe_huge = 772
scn_multi_taien = 773
scn_multi_al_thor_farm = 774
scn_multiplayer_maps_end = 775
scn_wedding = 776
scn_lair_steppe_bandits = 777
scn_lair_taiga_bandits = 778
scn_lair_desert_bandits = 779
scn_lair_forest_bandits = 780
scn_lair_mountain_bandits = 781
scn_lair_sea_raiders = 782
scn_quick_battle_scene_1 = 783
scn_quick_battle_scene_2 = 784
scn_quick_battle_scene_3 = 785
scn_quick_battle_emonds_field = 786
scn_quick_battle_shienaran_border_tower = 787
scn_quick_battle_scene_4 = 788
scn_quick_battle_scene_5 = 789
scn_quick_battle_maps_end = 790
scn_tutorial_training_ground = 791
scn_town_1_room = 792
scn_town_5_room = 793
scn_town_6_room = 794
scn_town_8_room = 795
scn_town_10_room = 796
scn_town_19_room = 797
scn_meeting_scene_steppe = 798
scn_meeting_scene_plain = 799
scn_meeting_scene_snow = 800
scn_meeting_scene_desert = 801
scn_meeting_scene_steppe_forest = 802
scn_meeting_scene_plain_forest = 803
scn_meeting_scene_snow_forest = 804
scn_meeting_scene_desert_forest = 805
scn_enterprise_tannery = 806
scn_enterprise_winery = 807
scn_enterprise_mill = 808
scn_enterprise_smithy = 809
scn_enterprise_dyeworks = 810
scn_enterprise_linen_weavery = 811
scn_enterprise_wool_weavery = 812
scn_enterprise_brewery = 813
scn_enterprise_oil_press = 814
scn_sea_b = 815
scn_sea_land = 816
scn_shienaran_border_tower_1 = 817
scn_al_thor_farm = 818
scn_emonds_field_raid_book_1 = 819
